Source/Shaders/Builtin/Functions/imageBasedLighting.glsl

```diff
@@ -0,0 +1,125 @@
+void proceduralIBL(
+  vec3 v,
+  vec3 n,
+  vec3 positionWC,
+  vec3 lightColorHdr,
+  czm_pbrParameters pbrParameters
+) {
```

Comment:
Style - does the curly bracket need to be on the next line?

Reply:
Hm, yeah I think it should.

Also... this IBL file shouldn't exist on this branch, I'll remove it